api/src/pcapi/routes/native/v1/serialization/bookings.py

```diff
@@ -31,6 +31,8 @@ class BookOfferResponse(BaseModel):
 
 class BookingVenueResponse(BaseModel):
     id: int
+    address: Optional[str]
+    postalCode: Optional[str]
```

Comment:
Est-ce qu'`Optional` est nécessaire ? Ces champs (tout comme  `city` et `publicName`, d'ailleurs) ne peuvent pas valoir `None` car, dans la base de données, la colonne est `NOT NULL`.

Reply:
J'ai regardé le model pour savoir s'il pouvait ne pas y avoir de valeur

J'ai vu `address IS NULL`

https://github.com/pass-culture/pass-culture-main/blob/22d722c684cbec0b74149b55dfd3df324884c2ec/api/src/pcapi/core/offerers/models.py#L58

et `postalCode` et `city` qui sont `nullable=True`

https://github.com/pass-culture/pass-culture-main/blob/22d722c684cbec0b74149b55dfd3df324884c2ec/api/src/pcapi/core/offerers/models.py#L151

https://github.com/pass-culture/pass-culture-main/blob/22d722c684cbec0b74149b55dfd3df324884c2ec/api/src/pcapi/core/offerers/models.py#L153

Je n'ai probablement pas regardé au bon endroit

Où as-tu trouvé que c'est `NOT NULL` ?